src/main/java/duke/CommandType.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+/**
+ * This is the enum class to store the types of
+ * commands a user can input to Duke
+ */
+public enum CommandType {
```

Comment:
Enum names are to be written in PascalCase

Reply:
"CommandType" is already in pascal case is it not?

Reply:
Oops, sorry I misread the line while I was reviewing it.